db/setup.sql

```diff
@@ -0,0 +1,20 @@
+-- CREATE TYPE org_status IF NOT EXISTS AS ENUM ('initial', 'allocated', 'connected');
+
+CREATE TABLE IF NOT EXISTS orgs (
+    id BIGSERIAL PRIMARY KEY,
+    org VARCHAR(42) NOT NULL UNIQUE,
+    expiry NUMERIC NOT NULL,
+    provider VARCHAR(20),
+    ip INET,
+    status VARCHAR(20) NOT NULL DEFAULT 'initial'
```

Comment:
Can we use an enum for this?

Reply:
Yes, using `varchar` temporarily to see what states we have.

Reply:
Will mark this as resolved for now, but will fix this eventually.

Reply:
https://github.com/radicle-dev/radicle-cloud/pull/1/commits/d678c7631ff32168b1b1bb4d974dcf0ba4bfabed